Updating_man

byteninjaa0 opened the pull request:
# SQUASH ME

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**
updated the man pages of rizin.1 and rz-diff.1
<!-- Explain the **details** for making this change. Is a new feature implemented? What existing problem does the pull request solve? How does the pull request solve these issues? Please provide enough information so that others can review your pull request. -->

...

**Test plan**

<!-- What steps should the reviewer take to test your pull request? Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots/videos. This is your time to re-check that everything works and that you covered all the edge cases -->

...

**Closing issues**

<!-- put "closes #XXXX" in your comment to auto-close the issue that your PR fixes (if any). -->

...

byteninjaa0 committed:
Updating_man

byteninjaa0 committed:
Merge branch 'dev' into Updating_man

wargio commented on binrz/man/rz-diff.1:
```diff
@@ -1,82 +1,82 @@
+              myers | Eugene W. Myers' O(ND) algorithm (no substitution)
+              leven | Levenshtein O(N^2) algorithm (with substitution)
+.It
+				     ssdeep | Context triggered piecewise hashing comparison
```
these spaces looks weird

byteninjaa0 committed:
removed extra spaces

byteninjaa0 committed:
Merge branch 'Updating_man' of https://github.com/sohail1734/rizin into Updating_man
removed spaces

XVilka commented on binrz/man/rz-diff.1:
```diff
@@ -1,82 +1,81 @@
-.Dd Feb 10, 2018
+.Dd  10, 2018
```
Broken date

XVilka commented on binrz/man/rizin.1:
```diff
@@ -1,132 +1,122 @@
-.Dd May 17, 2020
+.Dd 24 Aug, 2023
```
Please use the same format

XVilka commented on binrz/man/rizin.1:
```diff
@@ -1,132 +1,122 @@
+Forces the use of a specific rbin plugin.
+.It Fl h, hh
+Shows help messages, with -hh showing detailed help.
+.It Fl H ([var])
```
Probably worth noting that just `-H` without any argument prints useful information about the installation - location of various paths - config file, plugins, etc:
```
rizin -H
RZ_VERSION=0.7.0
RZ_PREFIX=/Users/user/.local
RZ_EXTRA_PREFIX=
RZ_MAGICPATH=/Users/user/.local/share/rizin/magic
RZ_INCDIR=/Users/user/.local/include/librz
RZ_LIBDIR=/Users/user/.local/lib
RZ_SIGDB=/Users/user/.local/share/rizin/sigdb
RZ_EXTRA_SIGDB=
RZ_LIBEXT=dylib
RZ_RCONFIGHOME=/Users/user/.config/rizin
RZ_RDATAHOME=/Users/user/.local/share/rizin
RZ_RCACHEHOME=/Users/user/.cache/rizin
RZ_LIBR_PLUGINS=/Users/user/.local/lib/rizin/plugins
RZ_EXTRA_PLUGINS=
RZ_USER_PLUGINS=/Users/user/.local/lib/rizin/plugins
RZ_IS_PORTABLE=0
```

XVilka commented on binrz/man/rizin.1:
```diff
@@ -1,132 +1,122 @@
 Do not load user settings/projects from ~/.rizinrc, ~/.config/rizin/rizinrc and the scripts inside .../rizinrc.d/ directory.
 .It Fl NN
-Same as -N but also disables the automatic loading of plugins on startup time
+Disables loading any scripts or plugins.
```
Please be consistent. Sometimes it `disable`, sometimes it `disables`. I suggest follow the same style everywhere.

byteninjaa0 committed:
Date updated
corrected -H command
Impllemented the same style throughout
added examples in rizin.1

wargio reviewed (approved):
LGTM besides an extra space.

wargio commented on binrz/man/rz-diff.1:
```diff
@@ -1,82 +1,87 @@
-.Dd Feb 10, 2018
+.Dd  Aug 25, 2023
```
extra useless space

XVilka committed:
Update rz-diff.1

XVilka committed:
Merge branch 'dev' into Updating_man
